### PR TITLE
hoogle: open up SSH access

### PR DESCRIPTION
**Pull request by garyverhaegen-da**
Required by security audit.

CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: infra/hoogle_server.tf [hunk 1]
```diff
@@ -18,6 +19,27 @@ resource "google_compute_firewall" "hoogle" {
...
 
+// Required only for security audit purposes
+resource "google_compute_firewall" "hoogle-ssh" {
+  name        = "hoogle-firewall-hole"
+  network     = "${google_compute_network.hoogle.name}"
+  target_tags = ["hoogle"]
+
+  source_ranges = [
+    // ???
+    "52.73.38.90/32",
```
Comment (cocreature) [inline infra/hoogle_server.tf:30]: @nycnewman What is this?

**Review by cocreature — APPROVED** [review]
Approving since this just makes the actual state match what we have. @nycnewman please add this machine to the ones that are controlled by terraform and _never_ make manual changes to it.

**Comment by nycnewman** [comment]
I'm proposing to roll back manual changes so suggest we do not proceed with this change.

**Comment by garyverhaegen-da** [comment]
> I'm proposing to roll back manual changes so suggest we do not proceed with this change.

Alright.

**Comment by garyverhaegen-da** [comment]
Can confirm network rules on hoogle machines is back to exactly what our Terraform files describe.

File: infra/hoogle_server.tf (hunk 1 above)
Comment (garyverhaegen-da) [reply infra/hoogle_server.tf:30]: Apparently that's the IP you'd get if you use the non-full-tunnel North Carolina VPN and (with a local config change) instruct your computer to route all traffic through it.

**Pull request closed without merging**